docs/docker.md

```diff
@@ -1,134 +1,151 @@
 # Running an epoch node on Docker
 
 This document describes:
+* [How to join the Roma network using Docker](#roma-network);
+* [How to use the Docker image](#docker-image);
 * [How to join the testnet using Docker](#testnet);
 * [How to run a local network using Docker](#localnet).
 
-You must have [docker installed](https://docs.docker.com/engine/installation/) on a host machine.
+You must have [docker installed](https://docs.docker.com/engine/installation/) on a host machine and **be familiar [how to operate docker containers](https://docs.docker.com)**.
```

Comment:
Ok, that disqualifies me :joy:

Reply:
It's more like a disclaimer that this document is not intended to explain how docker works and to operate it.